packages/paraview/env.sh

```diff
@@ -0,0 +1,22 @@
+if [ -z "$INSTALLER_PARAVIEW_ENV_SH" ]; then
+    INSTALLER_PARAVIEW_ENV_SH=1
+    installer_paraview_dir=$(readlink -f $(dirname $BASH_SOURCE))
+    . $installer_paraview_dir/../../env.sh
+
+    # modules
+    module load cmake
+
+    # Closed source, keep VESTEC only
+    visibility=private
```

Comment:
Is there any reason this is private?

Reply:
it can be public indeed
```suggestion
    visibility=public
```
just like that ?